src/scales/quantitative.js

```diff
@@ -198,10 +104,34 @@ export function ScaleLog(key, channels, {base = 10, domain = inferLogDomain(chan
   return ScaleQ(key, scaleLog().base(base), channels, {domain, ...options});
 }
 
+export function ScaleQuantile(key, channels, {
+  quantiles = 5,
+  scheme,
+  domain = inferFullDomain(channels),
+  range = ordinalScheme(scheme === undefined ? "rdylbu" : scheme)({length: quantiles}).slice(0, quantiles),
+  reverse,
+  ...options
+}) {
+  if (reverse) range = reverseof(range);
+  return ScaleQ(key, scaleQuantile(), [], {domain, range, ...options});
+}
+
 export function ScaleSymlog(key, channels, {constant = 1, ...options}) {
   return ScaleQ(key, scaleSymlog().constant(constant), channels, options);
 }
 
+export function ScaleThreshold(key, channels, {
+  domain = inferDomain(channels),
```

Comment:
A threshold scale’s domain represents the thresholds, so if we default to [min, max] here, we’ll have a 3-element range, but we’ll only use one of the three output colors (because all of the input values to the scale will necessarily be >=min and <=max). Probably a more sensible default would be [median] or [first quartile, median, third quartile]… or maybe we just have no default domain and require you to specify one, which might make the most sense for a threshold scale. If you want quantiles you can use a quantile scale.

Reply:
I decided that `domain = [0]` is a more reasonable default (better than crashing).